### DISCUSSION TITLE
i made a extenshion

### DISCUSSION THREAD

**Discussion by ThatLegend** [Prodigy-Hacking/ProdigyMathGameHacking#1214] [(A) Anything]
its for glitches in prodigy here is link https://chrome.google.com/webstore/detail/prodigy-math-game-hacking/jcmnbgkhaidhciannbmeieholahjibgc?hl=en&authuser=0

**Comment by donaldli2020** [comment]
You basically just used the old code.

**Comment by PatheticMustan** [comment]
this code not only has no relevant additions, it also uses a super outdated version that doesn't even work anymore. Why would you submit this?

**Comment by ThatLegend** [comment]
i just updated it

**Reply by donaldli2020** [reply]
wtf its still prodigyhacking.ml its still v 2.0.0

**Reply by donaldli2020** [reply]
& u didnt update
![image](https://user-images.githubusercontent.com/74021676/144334063-7170f3b4-8183-4fa9-a659-3cd9f2f20248.png)

**Comment by ThatLegend** [comment]
Yes, I did. it being reviewd by google

**Comment by 00100000** [comment]
Stealing this open source software, making pretty inconsequential changes, and re-releasing it under your own name is a pretty shameful thing to do. How can you talk about what "real hackers" do, when you leech off of other people's work and republish it @ThatLegend ?

**Comment by donaldli2020** [comment]
@ThatLegend You said you updated it but you didn't. It also looked like you got many accounts by yourself or friends to write positive reviews.
